### PR TITLE
Implements `git gud load next`

### LINKED ISSUE #133: `git gud load prev` & `git gud load previous`
Similar to #31, but we should be able to go backwards too. 

If you accidentally load the next level or just want to go back, this would be convenient

### PR DESCRIPTION
**Pull request by sahansk2**
Fixes #90 
fixes #133

### CODE REVIEW

**Comment by sahansk2** [comment]
Now fixes #133 as well.
I originally thought the code for loading previous/prev was going to be different, with an O(n) search for the skill/level names, but it ended up being much easier.

There's repeated code, so I'm going to make this cleaner.

File: gitgud/__main__.py [hunk 1]
```diff
@@ -304,14 +288,35 @@ def handle_load(self, args):
...
+                    level_to_load = level.prev_level
+                
+                if level_to_load is not None:
+                    self.load_level(level_to_load)
+                else:
+                    if query == "next":
+                        print_all_complete()
+                    else:
+                        print("Already on the first level; reloading first level...\n")
+                        self.load_level(all_skills["1"]["1"])
```
Comment (benthayer) [inline gitgud/__main__.py]: Can you remove this and change the print statement above to 
```
print('Already on the first level. To reload the current level, use "git gud reload"')
```

**Comment by sahansk2** [comment]
I've changed the print text, and how `git gud load prev/previous` works on the first level -- this PR should be good now.

**Comment by benthayer** [comment]
Great, I'll check it

**Pull request merged**